packages/api/helper/nominationGsheetToDB.js

```diff
@@ -70,7 +72,15 @@ module.exports = function gsheetToDB() {
               parseFloat(nomination[37].replace(/\$/g, '')) * 100
             ),
           },
-        });
+        }).then((array)=>{
+          //db.Nomination.findOrCreate returns a promise that when fullfilled returns an array with two elements 
+          //the first element is the nomination instance object that contains the dataValues among other things  
+          //the second element is a boolean that represents whether a nomination was created
+          isNewNomination = array[1]
```

Comment:
I don't think you need to declare the variable outside of here, can probably just use a local variable.